app/src/main/java/org/mozilla/focus/widget/InlineAutocompleteEditText.java

```diff
@@ -658,4 +641,24 @@ private void setIsKeyFromRemoteAppOrSoftKeyboardClear(final boolean isKeyFromRem
             removeAutocomplete(getText()); // Perhaps not strictly necessary.
         }
     }
+
+    @Override
+    public boolean dispatchKeyEvent(KeyEvent event) {
+        // This prevents the selector from entering the text area
+        final int keyCode = event.getKeyCode();
+        final boolean shouldDispatch = keyCode != KeyEvent.KEYCODE_DPAD_DOWN
+                || keyCode != KeyEvent.KEYCODE_DPAD_DOWN_LEFT
```

Comment:
nit: for future reference: || should be at the end of the line, not the beginning.